### ISSUE TITLE
Implicitly add Assume extension

### ISSUE THREAD

**Issue by asbjornu** [wjdp/htmltest#183]
**Is your feature request related to a problem? Please describe.**
I'm considering moving from HTMLProofer to HTMLTest due to its superior speed, but can't get htmltest to work for my Jekyll-built site since links to internal files omit the `.html` extension. In HTMLProofer, this is remedied by setting the [configuration](https://github.com/gjtorikian/html-proofer#configuration) property `assume_extension` to `.html`.

**Describe the solution you'd like**
It would be nice if htmltest, like HTMLProofer, was able to implicitly add an extension to links.

**Describe alternatives you've considered**
I did some searching to find similar issues and found #60 which is related, but not the same issue (and also not resolved).

**Comment by nickvigilante** [comment]
I agree. We've been told to use pretty URLs to help boost our site SEO.
